core/chains/evm/headtracker/orm.go

```diff
@@ -30,29 +28,42 @@ type ORM interface {
 	HeadByHash(ctx context.Context, hash common.Hash) (head *evmtypes.Head, err error)
 }
 
+var _ ORM = &orm{}
+
+// TODO: Set a reasonable timeout
+const defaultTimeout = 100 * time.Millisecond
```

Comment:
Noticed there's a TODO here. If this is the value we want to go with then this is just a nit to remove the comment

Reply:
Removed default timeouts from ORMs